recipe/bld.bat

```diff
@@ -1,3 +1,10 @@
+:: Move needed library files
+mkdir %LIBRARY_LIB%\graphviz
+xcopy /S %SRC_DIR%\lib\release\dll  %LIBRARY_LIB%\graphviz
```

Comment:
What dll's are copied in this operation?

Reply:
```
total 2.3M
-rwxr-xr-x 1 mel 1049089  34K Jan 17 21:35 Pathplan.dll*
-rwxr-xr-x 1 mel 1049089 192K Jan 17 21:35 ann.dll*
-rwxr-xr-x 1 mel 1049089  26K Jan 17 21:35 cdt.dll*
-rwxr-xr-x 1 mel 1049089  69K Jan 17 21:35 cgraph.dll*
-rwxr-xr-x 1 mel 1049089 532K Jan 17 21:35 gvc.dll*
-rwxr-xr-x 1 mel 1049089  94K Jan 17 21:35 gvplugin_core.dll*
-rwxr-xr-x 1 mel 1049089 208K Jan 17 21:35 gvplugin_dot_layout.dll*
-rwxr-xr-x 1 mel 1049089 257K Jan 17 21:35 gvplugin_gd.dll*
-rwxr-xr-x 1 mel 1049089  45K Jan 17 21:35 gvplugin_gdiplus.dll*
-rwxr-xr-x 1 mel 1049089 737K Jan 17 21:36 gvplugin_neato_layout.dll*
-rwxr-xr-x 1 mel 1049089  31K Jan 17 21:36 gvplugin_pango.dll*
-rwxr-xr-x 1 mel 1049089  42K Jan 17 21:35 vmalloc.dll*
```

Reply:
Truth be told,  I don't know exactly if they are needed, but right now I can't  create a windows conda-forge recipe for `pygraphviz` since it needs a link to the `include-path` and `library-path` when running `python setup.py install`. See [this link](http://pygraphviz.github.io/documentation/pygraphviz-1.5/install.html)

Reply:
The corresponding `*.so` files seemed to be in the Linux packages. So I thought these better be included as well.

Reply:
The problem i see with this is that some dll's (like `vmalloc.dll`) could conflict with other packages.

@melund, can you assure none of these dll's conflict with other conda-forge packages?

Reply:
@ccordoba12, not sure exactly how I could be assured of that. They are placed in a separate `graphviz` subfolder. Is that enough? But I don't know if there could be conflicts when the DLL's are loaded by other packages.

Reply:
> They are placed in a separate graphviz subfolder

Right, sorry, I didn't saw that. Then this PR is fine for me.